CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ While some features like the clipboard, menus or file dialogs are not yet availa
 - X11: Support `Application::quit`. ([#900] by [@xStrom])
 - GTK: Support file filters in open/save dialogs. ([#903] by [@jneem])
 - X11: Support key and mouse button state. ([#920] by [@jneem])
+- Routing `Event::FocusChanged` for descendant widgets. ([#925] by [@yrns])
```

Comment:
~Event::FocusChanged for~ LifeCycle::FocusChanged to

Also define your username link at the bottom where all the other username links are.